src/__test__/sum.test.js

```diff
@@ -0,0 +1,5 @@
+import sum from '../sum.js';
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
```

Comment:
- [ ] I really don't know what this test is for, kindly remove it

Reply:
okay sir